tests/conftest.py

```diff
@@ -38,6 +38,15 @@ def session():
     Base.metadata.drop_all(bind=engine)
 
 
+@pytest.fixture
+def test_session():
```

Comment:
Pull latest changes, and then you can discard this fixture :)

Reply:
Awesome, thank you!